src/main/java/com/ericsson/gerrit/plugins/eiffel/configuration/EiffelPluginConfiguration.java

```diff
@@ -109,4 +112,12 @@ public boolean isEnabled() {
     public String getFlowContext() {
         return flowContext;
     }
+
+    public File getPluginDir(File pluginDir) {
```

Comment:
I personally prefer having the full words written out instead of shortened like this, so in this case I would prefer naming the method "getPluginDirectory". And add some javadoc explaining that the directory is the location for the configurations, because it is not apparent from the method name only.

Reply:
It is not the location of configurations.

Reply:
Alright then, even more reason to maybe add javadoc for what this directory is used for, since my guess was wrong :)

Reply:
Done.